src/main/proto/ga4gh/rna_quantification_service.proto

```diff
@@ -0,0 +1,147 @@
+syntax = "proto3";
+
+package ga4gh;
+
+import "ga4gh/rna_quantification.proto";
+
+service RnaQuantificationService {
+// Gets a list of 'RnaQuantification' matching the search criteria.
+
+// 'POST /rnaquantification/search' must accept JSON version of
+// 'SearchRnaQuantificationRequest' as the post body and will return a JSON
+// version of 'SearchRnaQuantificationResponse'.
+rpc SearchRnaQuantifications(SearchRnaQuantificationsRequest)
+      returns (SearchRnaQuantificationsResponse);
+
+
+// Gets a `RnaQuantification` by ID.
+// `GET /rnaquantification/{id}` will return a JSON version of
+// `RnaQuantification`.
+rpc GetRnaQuantification(GetRnaQuantificationRequest)
+      returns (RnaQuantification);
+
+// Gets a list of 'ExpressionLevel' matching the search criteria.
+
+// 'POST /expressionlevel/search' must accept JSON version of
+// 'SearchExpressionLevelRequest' as the post body and will return a JSON
+// version of 'SearchExpressionLevelResponse'.
+rpc SearchExpressionLevels(SearchExpressionLevelsRequest)
+      returns (SearchExpressionLevelsResponse);
+
+
+// Gets a list of 'QuantificationGroup' matching the search criteria.
+
+// 'POST /quantificationgroup/search' must accept JSON version of
+// 'SearchQuantificationGroupRequest' as the post body and will return a JSON
+// version of 'SearchQuantificationGroupResponse'.
+rpc SearchQuantificationGroups(SearchQuantificationGroupsRequest)
+      returns (SearchQuantificationGroupsResponse);
+}
+
+// ****************  /rnaquantification/search  *******************
+
+// This request maps to the body of 'POST /rnaquantification/search'
+// as JSON.
+message SearchRnaQuantificationsRequest {
+  // If present, return only Rna Quantifications which belong to this set.
+  string rna_quantification_id = 1;
```

Comment:
Not sure I follow what this ``rna_quantification_id`` refers to. You mention a set in the comments?